### ISSUE TITLE
pre-commit hook

### ISSUE THREAD

**Issue by gnuletik** [santhosh-tekuri/jsonschema#108]
Hi,

Are you open to provide a `.pre-commit-hooks.yaml` to use the `jv` CLI inside `pre-commit` ?

Thanks !

**Comment by santhosh-tekuri** [comment]
I am not sure of how to do it. If you like to contribute, I will be happy

**Comment by gnuletik** [comment]
Cool ! I don't have much time now but I'll check if the subject has attraction.

There is this PR from golangci-lint repo that can be used as an example : https://github.com/golangci/golangci-lint/pull/311/files
But I'm not sure how pre-commit is able to find the binary.

**Comment by jlucktay** [comment]
From [the pre-commit docs here](https://pre-commit.com/index.html#golang):
> The hook repository must contain go source code. It will be installed via go `install ./...`.
> pre-commit will create an isolated `GOPATH` for each hook and the [`entry`](https://pre-commit.com/index.html#hooks-entry) should match an executable which will get installed into the `GOPATH`'s `bin` directory.

When I clone this repo and do `go install ./...` from the root, nothing new shows up in my `$GOBIN` directory.
This seems to be because `cmd/jv/go.mod` exists and the `main.go` file there in that directory is inside a different Go module.

With that `cmd/jv/go.mod` removed, the barebones `.pre-commit-hooks.yaml` file would start from something like this:

```yaml
id: jsonschema-validate
name: Validate JSON against JSON Schema
entry: jv
language: golang
```

More details on fields for the above YAML are [here](https://pre-commit.com/index.html#creating-new-hooks).

**Comment by santhosh-tekuri** [comment]
fixed in v6.0.0
